src/purify.js

```diff
@@ -130,11 +128,8 @@ function createDOMPurify(window = getGlobal()) {
     }
   }
 
-  let trustedTypesPolicy = _createTrustedTypesPolicy(
-    trustedTypes,
-    originalDocument
-  );
-  let emptyHTML = trustedTypesPolicy ? trustedTypesPolicy.createHTML('') : '';
+  let trustedTypesPolicy;
```

Comment:
nit: I would move `let trustedTypesPolicy` declaration just before `if (cfg.TRUSTED_TYPES_POLICY) {` line

Reply:
that wouldn't be possible in the present form of the code, the `if` condition is part of the `_parseConfig` function and `let trustedTypesPolicy` is in the upper scope. Moving it before the `if` would make it a local variable to `_parseConfig` and inaccessible to the rest of the methods that look for it on the upper scope. This would be a move involved effort.

Reply:
Let's keep current change then if it's intentional.